app/models/review.rb

```diff
@@ -6,5 +6,6 @@ class Review < ActiveRecord::Base
   # has_many :likes, dependent: :destroy
 
   validates :user, presence: true
-
+  validates :rating, presence: true,
+            numericality: { minimum: RATINGS[0], maximum: RATINGS[-1] }
```

Comment:
You might want to just hardcode RATINGS[0] and RATINGS[-1] in here. I don't know if it goes from 0 to 20 or 1 to 8, etc. by just looking at this model.